mirdata/utils.py

```diff
@@ -29,11 +29,45 @@
 import hashlib
 import os
 import json
+import types
 
 
 MIR_DATASETS_DIR = os.path.join(os.getenv('HOME', '/tmp'), 'mir_datasets')
 
 
+class Track(object):
+    def __repr__(self):
+        properties = [v for v in dir(self.__class__) if not v.startswith('_')]
+        attributes = [
+            v for v in dir(self) if not v.startswith('_') and v not in properties
+        ]
+
+        repr_str = "Track(\n"
+
+        for attr in attributes:
+            val = getattr(self, attr)
+            if isinstance(val, str):
+                val = '"{}"'.format(val)
+            repr_str += "  {}={},\n".format(attr, val)
+
+        for prop in properties:
+            val = getattr(self.__class__, prop)
+            if isinstance(val, types.FunctionType):
+                continue
+
+            if val.__doc__ is None:
+                raise ValueError("{} has no documentation".format(prop))
+
+            val_type_str = val.__doc__.split(':')[0]
```

Comment:
do you validate that docstrings are correctly formatted?

Reply:
nope - should I? if it doesn't have a `:` character somewhere it'll just print the whole docstring, which is ugly but not broken